docs/includes/powerpoint-preview.md

```diff
@@ -1,52 +1,16 @@
 | Class | Fields | Description |
 |:---|:---|:---|
-|[BulletFormat](/javascript/api/powerpoint/powerpoint.bulletformat)|[visible](/javascript/api/powerpoint/powerpoint.bulletformat#powerpoint-powerpoint-bulletformat-visible-member)|Specifies if the bullets in the paragraph are visible.|
-|[ParagraphFormat](/javascript/api/powerpoint/powerpoint.paragraphformat)|[bulletFormat](/javascript/api/powerpoint/powerpoint.paragraphformat#powerpoint-powerpoint-paragraphformat-bulletformat-member)|Represents the bullet format of the paragraph.|
-||[horizontalAlignment](/javascript/api/powerpoint/powerpoint.paragraphformat#powerpoint-powerpoint-paragraphformat-horizontalalignment-member)|Represents the horizontal alignment of the paragraph.|
 |[Presentation](/javascript/api/powerpoint/powerpoint.presentation)|[getSelectedShapes()](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-getselectedshapes-member(1))|Returns the selected shapes in the current slide of the presentation.|
 ||[getSelectedSlides()](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-getselectedslides-member(1))|Returns the selected slides in the current view of the presentation.|
 ||[getSelectedTextRange()](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-getselectedtextrange-member(1))|Returns the selected {@link PowerPoint.TextRange} in the current view of the presentation.|
 ||[getSelectedTextRangeOrNullObject()](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-getselectedtextrangeornullobject-member(1))|Returns the selected {@link PowerPoint.TextRange} in the current view of the presentation.|
 ||[setSelectedSlides(slideIds: string[])](/javascript/api/powerpoint/powerpoint.presentation#powerpoint-powerpoint-presentation-setselectedslides-member(1))|Selects the slides in the current view of the presentation.|
-|[Shape](/javascript/api/powerpoint/powerpoint.shape)|[fill](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-fill-member)|Returns the fill formatting of this shape.|
-||[getParentSlide()](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-getparentslide-member(1))|Returns the parent {@link PowerPoint.Slide} object that holds this `Shape`.|
+|[Shape](/javascript/api/powerpoint/powerpoint.shape)|[getParentSlide()](/javascript/api/powerpoint/powerpoint.shape#powerpoint-powerpoint-shape-getparentslide-member(1))|Returns the parent {@link PowerPoint.Slide} object that holds this `Shape`.|
```

Comment:
doesn't seem like the @ links didn't resolve

Reply:
Hmmm. Existing docs are like this too. I've started an email thread.